syslog-ng-helm-chart/values.yaml

```diff
@@ -0,0 +1,107 @@
+# Default values for syslog-ng.
+
+replicaCount: 1
+
+image:
+  repository: balabit/syslog-ng
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "3.27.1"
+
+# By default, the network() driver binds to 0.0.0.0, meaning that it listens 
+# on every available IPV4 interface on the TCP/601 port.
+
+config: |
+  @version: 3.27
+  @include "scl.conf"
+  options {
+    # enable or disable directory creation for destination files
+    create_dirs(yes);
+
+    # keep hostnames from source host
+    keep_hostname(yes);
+
+    # use ISO8601 timestamps
+    ts_format(iso);
+  };
+  log {
+  	source {
+  		network();
```

Comment:
Could we perhaps use default-network-drivers() here?

That would enable automatic parsing of a number of sources using the app-parser() framework. It also opens up a few more ports (e.g. tls enabled stuff).

Tls support would be useful, e.g. the management of the x.509 certificates.

Reply:
that requieres another iteration on this chart, maybe next version of this. I suggest to include this as it is now, and let the community start pushing code for all the features are missing here.